packages/vm/package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "@oasislabs/ethereumjs-vm",
   "version": "6.0.0-rc.1",
-  "description": "An Ethereum VM implementation",
+  "description": "A Sapphire-compatible Ethereum VM implementation",
   "keywords": [
     "ethereum",
+    "sapphire",
+    "oasis",
     "VM"
   ],
-  "homepage": "https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/vm#readme",
+  "homepage": "https://github.com/oasislabs/NomicFoundation-ethereumjs-vm/tree/master/packages/vm",
   "bugs": {
-    "url": "https://github.com/ethereumjs/ethereumjs-monorepo/issues?q=is%3Aissue+label%3A%22package%3A+vm%22"
+    "url": "https://github.com/oasislabs/NomicFoundation-ethereumjs-vm/issues"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ethereumjs/ethereumjs-monorepo.git"
+    "url": "https://github.com/oasislabs/NomicFoundation-ethereumjs-vm.git"
   },
   "license": "MPL-2.0",
   "author": "mjbecze <mjbecze@gmail.com>",
```

Comment:
should we change the author fields or just remove them?

Reply:
You can leave the author field in, as it's polite to cite the author.